clang/lib/CConv/CheckedRegions.cpp

```diff
@@ -303,6 +309,8 @@ bool CheckedRegionFinder::isInStatementPosition(CallExpr *C) {
   } else {
     //TODO there are other statement positions
     //     besides child of compound stmt
+    auto PSL = PersistentSourceLoc::mkPSL(C, *Context);
+    emitCauseDiagnostic(&PSL);
```

Comment:
I take it that you are identifying variadic calls somehow, and keeping track of them as a root cause? How can you tell that this call is variadic? Also, for the test you added, it seems that variadic calls are surrounded by `_Unchecked { ... }` -- will this always happen, and if so what's the effect on this change?

Reply:
Variadic calls should be surrounded w/ unchecked if two conditions are met:
1) They are in an otherwise checked region
2) They are not the only statement in the region
However, there are going to variadic calls that can't be surrounded, for example a call that's part of an expression (like `if(sscanf(.....))` and that's what will cause this warning to fire.